release_notes.md

```diff
@@ -0,0 +1,6 @@
+# Fix Details
+
+- Removed `Scenario` module and SVT from the solution pack
+- Demo alert is now created with type `Suspicious Email` instead of `Phishing`
```

Comment:
Corrected demo alert type from`Phishing` to `Suspicious Email`

Reply:
Made suggested changes in release notes